src/libpsl-native/src/getuserfrompid.cpp

```diff
@@ -45,6 +49,22 @@ char* GetUserFromPid(pid_t pid)
 
     return GetPwUid(oldp.kp_eproc.e_ucred.cr_uid);
 
+#elif defined(__FreeBSD__)
```

Comment:
Instead of this can the following work?

```C
#elif (defined(__APPLE__) && defined(__MACH__)) || defined(__FreeBSD__)
```

Reply:
The return statement is the only one that really changes from macOS' implementation. I rewrote the ifdefs to replace only the return section.